inference-engine/tests/functional/plugin/cpu/fusing_tests/src/group_convolution_fusing.cpp

```diff
@@ -0,0 +1,246 @@
+    } else {
+        function = makeNgraphFunction(ngPrc, params, groupConv, postNodes);
+    }
+}
+
+TEST_P(GroupConvolutionLayerFusingTest, CompareWithRefs) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+
+    Run();
+    CPUTestUtils::CheckCPUImpl(executableNetwork, "Convolution", inFmts, outFmts, selectedType);
+    FusingTestUtils::CheckFusing(executableNetwork, "Convolution", fusedOps);
+}
+
+namespace {
+
+std::vector<fusingSpecificParams> fusingParamsSet {
+    // activations
+    fusingRelu,
+    fusingElu,
+    fusingSigmoid,
+    fusingClamp,
+    fusingPRelu,
+    // other patterns
+    fusingReluScaleShift,
+    fusingFakeQuantizePerChannelRelu,
+    fusingSum,
+    fusingSwishPattern,
+};
+
+/* INSTANCES */
+/* ============= GroupConvolution (Planar 2D) ============= */
+const auto groupConvParams_ExplicitPadding_Planar_2D = ::testing::Combine(
+        ::testing::Values(SizeVector({3, 3})),           //  kernels
+        ::testing::Values(SizeVector{1, 1}),             //  strides
+        ::testing::Values(std::vector<ptrdiff_t>{0, 0}), //  padBegins
+        ::testing::Values(std::vector<ptrdiff_t>{0, 0}), //  padEnds
+        ::testing::Values(SizeVector{1, 1}),             //  dilations
+        ::testing::Values(4),                            //  numOutChannels
+        ::testing::Values(2),                            //  numGroups
+        ::testing::Values(ngraph::op::PadType::EXPLICIT)
+);
+
+INSTANTIATE_TEST_CASE_P(GroupConvFusing_Planar_2D, GroupConvolutionLayerFusingTest,
+                        ::testing::Combine(
+                                ::testing::Combine(
+                                        groupConvParams_ExplicitPadding_Planar_2D,
+                                        ::testing::Values(Precision::FP32),
+                                        ::testing::Values(std::vector<size_t >{1, 10, 5, 5}),
+                                        ::testing::Values(CommonTestUtils::DEVICE_CPU)),
+                                ::testing::ValuesIn(filterCPUInfoForDevice(CPUParams_Planar_2D)),
+                                ::testing::ValuesIn(fusingParamsSet)),
+                        GroupConvolutionLayerFusingTest::getTestCaseName);
+
+/* ============= GroupConvolution (Planar 3D) ============= */
+const auto groupConvParams_ExplicitPadding_Planar_3D = ::testing::Combine(
+        ::testing::Values(SizeVector({3, 3, 3})),           //  kernels
+        ::testing::Values(SizeVector{1, 1, 1}),             //  strides
+        ::testing::Values(std::vector<ptrdiff_t>{0, 0, 0}), //  padBegins
+        ::testing::Values(std::vector<ptrdiff_t>{0, 0, 0}), //  padEnds
+        ::testing::Values(SizeVector{1, 1, 1}),             //  dilations
+        ::testing::Values(4),                               //  numOutChannels
+        ::testing::Values(2),                               //  numGroups
+        ::testing::Values(ngraph::op::PadType::EXPLICIT)
+);
```

Comment:
You construct a static single element here. You no need in call of `Combine` generator here.
All arguments is "::testing::Values()" with single argument.